.github/scripts/validate_scale_config.py

```diff
@@ -98,47 +137,83 @@ def runner_types_are_equivalent(
 
 
 def is_config_consistent_internally(runner_types: Dict[str, Dict[str, str]]) -> bool:
-    """
-    Ensure that for every linux runner type in the config, there is a corresponding runner type with the
-    prefix "amz2023." that contains all the same settings except for the ami
+    f"""
+    Ensure that for every linux runner type in the config:
+
+    1 - they match RunnerTypeScaleConfig https://github.com/pytorch/test-infra/blob/main/terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/runners.ts#L50
```

Comment:
consider changing this to point to a specific commit so that the line number doesn't go out of date